tt-metal-yolov4/README.md

```diff
@@ -0,0 +1,41 @@
+# TT Metalium YoloV4 Inference API
```

Comment:
Nit-picking this one, should we change to "YOLOv4"?

Reply:
Ageed, changed in https://github.com/tenstorrent/tt-inference-server/pull/52/commits/4c0a27738c78837bea449291e75330d4b3cdebed